assets/app/view/game/buy_sell_shares.rb

```diff
@@ -161,6 +161,7 @@ def render_share_exchange(shares, company, source: 'Market')
 
       def render_merge
         return [] unless @step.current_actions.include?('choose')
+        return [] unless @step.respond_to?(:can_merge)
```

Comment:
Do you need a question mark at the end of :can_merge? to match the method in question on line 165?

Reply:
Tried it locally and it worked on the broken game, so I guess not.

Reply:
Recommend changing it to include the question mark, all the same.

Reply:
> Tried it locally and it worked on the broken game, so I guess not.

I'm guessing this is caused by however Opal handles function names with question marks, since you can't have that in JS. Definitely better to actually have it though